sdk/communication/azure-communication-phonenumbers/README.md

```diff
@@ -16,153 +16,129 @@ Install the Azure Communication Phone Numbers client library for Python with [pi
 pip install azure-communication-phonenumbers
 ```
 
-# Key concepts
+## Key concepts
 
-## CommunicationPhoneNumberClient
 ### Initializing Phone Numbers Client
 ```python
-# You can find your endpoint and access token from your resource in the Azure Portal
+# You can find your connection string from your resource in the Azure Portal
 import os
-from azure.communication.phonenumbers import PhoneNumbersAdministrationClient
+from azure.communication.phonenumbers import PhoneNumbersClient
 from azure.identity import DefaultAzureCredential
 
-endpoint = os.getenv('AZURE_COMMUNICATION_SERVICE_ENDPOINT')
+endpoint = os.getenv('AZURE_COMMUNICATION_SERVICE_CONNECTION_STRING')
 
 # To use Azure Active Directory Authentication (DefaultAzureCredential) make sure to have your
 # AZURE_TENANT_ID, AZURE_CLIENT_ID and AZURE_CLIENT_SECRET as env variables.
 phone_number_administration_client = PhoneNumbersAdministrationClient(endpoint, DefaultAzureCredential())
 
 ```
-### Initializing Phone Numbers Client Using Connection String
+### Initializing the Client Using Your Connection String
 Connection string authentication is also available for Phone Numbers Client.
 
 ```python
-# You can find your endpoint and access token from your resource in the Azure Portal
+# You can find your connection string from your resource in the Azure Portal
 import os
-from azure.communication.phonenumbers import PhoneNumbersAdministrationClient
+from azure.communication.phonenumbers import PhoneNumbersClient
 
 connection_str = os.getenv('AZURE_COMMUNICATION_SERVICE_CONNECTION_STRING')
 phone_number_administration_client = PhoneNumbersAdministrationClient.from_connection_string(connection_str)
 ```
 
-### Phone plans overview
+### Phone Number Types overview
 
-Phone plans come in two types; Geographic and Toll-Free. Geographic phone plans are phone plans associated with a location, whose phone numbers' area codes are associated with the area code of a geographic location. Toll-Free phone plans are phone plans not associated location. For example, in the US, toll-free numbers can come with area codes such as 800 or 888.
+Phone numbers come in two types; Geographic and Toll-Free. Geographic phone plans are phone plans associated with a location, whose phone numbers' area codes are associated with the area code of a geographic location. Toll-Free phone plans are phone plans not associated location. For example, in the US, toll-free numbers can come with area codes such as 800 or 888.
```

Comment:
still refers to phone plans after the first change to phone numbers. is that correct?